components/financial_assistance/app/views/financial_assistance/applications/submit_your_application.html.erb

```diff
@@ -98,7 +98,7 @@
       </div>
       <div class="col-md-4 px-0">
         <label for="last_name_thank_you" class="required"><%= l10n("last_name") %></label>
-        <%= text_field_tag "last_name_thank_you", nil, placeholder: l10n("last_name"), required: false, class: "thank_you_field w-100", id: "last_name_thank_you" %>
+        <%= text_field_tag "last_name_thank_you", nil, placeholder: l10n("last_name"), required: true, class: "thank_you_field w-100", id: "last_name_thank_you" %>
```

Comment:
Does this also cover filling the field with something that doesn't match their last name?

Reply:
So it will be covered in a different PR?

Reply:
I only wanted to ask because I believe this field may still be missing that validation

Reply:
Fix is in, checking against a pattern